lms/templates/problem_notifications.html

```diff
@@ -2,15 +2,15 @@
                                    notification_message, should_enable_next_hint"/>
 <%! from django.utils.translation import ugettext as _ %>
 
-<div class="notification ${notification_type} ${'notification-'}${notification_name} ${'is-hidden' if notification_name == 'gentle-alert' else '' }"
+<div class="notification ${notification_type} ${'notification-'}${notification_name}
+      ${'is-hidden' if notification_name == 'gentle-alert' or notification_name == 'save' or notification_name == 'hint' else '' }"
```

Comment:
Nit: How about you reverse this so 'is-hidden' is the default (which will make the statement shorter)?